CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
+* Cluster Autoscaler will delete nodes that encounter provisioning errors/ failures immediately, instead of waiting for the full max-node-provision-time defined in the [cluster autoscaler profile](https://learn.microsoft.com/azure/aks/cluster-autoscaler?tabs=azure-cli#cluster-autoscaler-profile-settings). This change significantly reduces scale-up delays caused by failed node provisioning attempts.
+* AKS Automatic clusters can now only be created with the `stable` upgrade channel and the `NodeImage` Node OS upgrade channel. Existing clusters are not affected.
+* [Node Auto Provisioning](https://learn.microsoft.com/azure/aks/node-autoprovision) default AKSNodeClasses will now use Ubuntu 22.04 for Kubernetes versions < 1.34 and Ubuntu 24.04 for Kubernetes versions 1.34+. This ensures consistency across AKS node image defaults. This does not affect existing clusters' default AKSNodeClasses.
+* The hostpath volume Validation Admission Policy (VAP) is now more flexible. An explicit allowlist of container images is now allowed to mount a hostpath volume, rather than blanket denying pods that specify a hostpath volume. The allowlist includes fluent-bit images from the `mcr.microsoft.com/oss` path and Geneva mdsd images used by the Azure Backup extension. Additional system-excluded namespaces include: `azappconfig-system`, `azureml`, `dapr-system`, `dataprotection-microsoft`, `flux-system`, `acstor`, `sc-system`, and `azure-extensions-usage-system`.
+* Starting with Kubernetes version 1.33, clusters using Azure CNI Powered by Cilium will include a new AKS-managed component named iptables monitor. This component is a sidecar container alongside the Cilium agent that will set labels on the Cilium node if a user iptables rule is detected.
+* The metric `ingress_upstream_latency_seconds` has been [removed](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.12.0) following its deprecation upstream.
+* `InPlaceOrRecreate` feature gate enabled for VPA(Vertical Pod Autoscaler) 1.4.2 in AKS 1.34, allowing customers to use the `InPlaceOrRecreate` update mode in their VPA objects. The feature is alpha in VPA `1.4.2` and must be explicitly enabled; it does not change the default update mode (auto), so only customers who opt in by specifying `InPlaceOrRecreate` will be affected. For more details on the feature: [Official feature doc](https://kubernetes.io/blog/2025/05/16/kubernetes-v1-33-in-place-pod-resize-beta/), [Upstream release note](https://github.com/kubernetes/autoscaler/releases/tag/vertical-pod-autoscaler-1.4.0).
+* Azure CNI and CNS have been updated to version [1.7.4](https://github.com/Azure/azure-container-networking/releases/tag/v1.7.4) with behavior change of Pod Subnet * Dynamic IP Allocation will SNAT Azure DNS traffic (168.63.129.16) using the node IP instead of the pod CIDR IP, aligning the behavior with Static Block Allocation.
```

Comment:
- can we elaborate on the behavior change for first point on pod subnet
- on second point
  - there's a missing newline
  - any docs link?

Reply:
@chasewilson is there a doc link for "Pod Subnet - Dynamic IP Allocation will SNAT Azure DNS traffic (168.63.129.16) using the node IP instead of the pod CIDR IP, aligning the behavior with Static Block Allocation."